chamge/arbiter.c

```diff
@@ -1,8 +1,3 @@
-/**
- *  Copyright 2019 SK Telecom, Co., Ltd.
- *    Author: Jeongseok Kim <jeongseok.kim@sk.com>
- *
- */
 
```

Comment:
You may forget to add the license to this file.

Reply:
Oops.. T.T You have eagle eye and I am heodang.